internal/util/netmsg.go

```diff
@@ -3,12 +3,19 @@ package util
 import (
 	"bytes"
 	"encoding/binary"
+	"fmt"
 	"net"
 )
 
-const RESPONSE_BUFFER_SIZE = 8
+const (
+	MAX_FORM_MEM_SIZE_BYTES = 10 << 20 // 20 MB
+	RESPONSE_BUFFER_SIZE    = 8
+)
 
-func ReadFromUnixSocket(connection net.Conn) ([]byte, error) {
+// Read a message from a network connection.
+// First 8 bytes: the size of the message being read.
```

Comment:
This needs to be a tab bit more precise.
Specifically, you don't say if the initial size includes the 8 bytes for the size itself.

Also, I'm not a fan of the colons here.
I would go with something like: "The first 8 bytes is ..., and the remaining content should be x bytes of ... .".